src/features/settings/general/comments/DownvotedCollapsedByDefault.tsx

```diff
@@ -0,0 +1,25 @@
+import { IonItem, IonToggle } from "@ionic/react";
+
+import { useAppDispatch, useAppSelector } from "#/store";
+
+import { setDownvotedCollapsedByDefault } from "../../settingsSlice";
+
+export default function DownvotedCollapsedByDefault() {
+  const dispatch = useAppDispatch();
+  const downvotedCollapsedByDefault = useAppSelector(
+    (state) => state.settings.general.comments.downvotedCollapsedByDefault,
+  );
+
+  return (
+    <IonItem>
+      <IonToggle
+        checked={downvotedCollapsedByDefault}
+        onIonChange={(e) =>
+          dispatch(setDownvotedCollapsedByDefault(e.detail.checked))
+        }
+      >
+        Collapse Downvoted Comments
```

Comment:
I was initially confused by this! I thought it would collapse when I downvote. Instead, it collapses comments with negative cumulative vote (which makes more sense!).

I think we should come up with a different setting that avoids this confusion. Some options...

as a switch, "Auto Collapse Negative Votes"

or, as a dropdown with options:

"Collapse Downvoted Comments..."

- "When Negative"
- "When Highly Downvoted" (some cool algorithm we build hehe)
- "Never" (default)

I personally prefer the latter, and it opens the door for more customization. What are your thoughts?